### DISCUSSION TITLE
Currenxie Payout

### DISCUSSION THREAD

**Discussion by prakpk** [juspay/hyperswitch#3737] [Ideas / Feature Requests]
Hello, I would like to request for a feature for a payout module catering to global banking provider Currenxie. 
Link: https://www.currenxie.com/

**Comment by AnandKGanesh** [comment]
Thanks for the suggestion.

Could you please elaborate on your need for this provider and what the use case is?

Please note that our bandwidth for Q1 is allocated to projects predefined in our roadmap. We'll review this at the end of Q1 and keep you posted on updates here.

**Reply by prakpk** [reply]
Hello Anand, 
Apologies for the late reply. Currenxie is a global banking solution. It will add depth, provide more flexibility to the payout section, and enable us to provide payout capabilities to more regions. Adyen and Wise add limited depth to the payout section. Having other BaaS sections will enable a wider appeal to the payout section this payout forms a vital part of any payment processing flow. Our use case is we want to have global banking on our platform alongside Payment Orchestration and having this integration will ensure we can easily enable a stronger integration.
